packages/block-editor/README.md

```diff
@@ -776,6 +776,8 @@ _Properties_
 -   _\_\_experimentalBlockDirectory_ `boolean`: Whether the user has enabled the Block Directory
 -   _\_\_experimentalBlockPatterns_ `Array`: Array of objects representing the block patterns
 -   _\_\_experimentalBlockPatternCategories_ `Array`: Array of objects representing the block pattern categories
+-   _\_\_experimentalAvailableImageSizes_ `Array`: Array of available image sizes.
```

Comment:
Also it seems we already have a `imageSizes` setting in `block-editor` how is this one different?

Reply:
`imageSizes` is just a (hardcoded?) list of image size names, whereas this contains all the information about the image size as passed to [`add_image_size()`](https://developer.wordpress.org/reference/functions/add_image_size/) in WordPress, i.e. width, height, and whether it's a hard crop or soft crop.

The TypeScript type is as follows

```
export type ImageSizeCrop = {
	name?: string;
	width: number;
	height: number;
	crop?:
		| boolean
		| [ 'left' | 'center' | 'right', 'top' | 'center' | 'bottom' ];
};
```